plugin/core/views.py

```diff
@@ -270,6 +270,8 @@ def versioned_text_document_identifier(view: sublime.View, version: int) -> Dict
 def text_document_position_params(view: sublime.View, location: int) -> TextDocumentPositionParams:
     return {"textDocument": text_document_identifier(view), "position": position(view, location)}
 
+def text_document_range_params(view: sublime.View, region: sublime.Region) -> Dict[str, Any]:
+    return {"textDocument": text_document_identifier(view), "range": region_to_range(view, region).to_lsp()}
```

Comment:
Did scala implementation change and now it accepts range in `range` parameter?
Also, if that's the case, you probably could also send `position` per normal?

Reply:
Only range or position should be sent https://scalameta.org/metals/docs/integrations/new-editor#textdocumenthover

Reply:
OK.
I would think that having data on where the cursor is exactly could be useful but it's up to their implementation...